README.md

```diff
@@ -313,39 +313,47 @@ pipenv run pybabel update -i app/translations/messages.pot -d app/translations
+| EQ_RABBITMQ_QUEUE_NAME                    | submit_q          | The name of the submission queue                                                              |
+| EQ_RABBITMQ_ENABLED                       | True              |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_DRIVER    | postgresql        |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_HOST      |                   |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_PORT      | 5432              |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_DATABASE_NAME      | digitaleqrds      |                                                                                               |
+| EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS | 10000             |                                                                                               |
+| EQ_STORAGE_BACKEND                        | datastore         |                                                                                               |
+| EQ_DATASTORE_PROJECT_ID                   | datastore         |                                                                                               |
+| EQ_DATASTORE_EMULATOR_CREDENTIALS         | False             |                                                                                               |
+| EQ_DYNAMODB_ENDPOINT                      |                   |                                                                                               |
+| EQ_DYNAMODB_MAX_RETRIES                   | 5                 |                                                                                               |
+| EQ_DYNAMODB_MAX_POOL_CONNECTIONS          | 30                |                                                                                               |
+| EQ_SUBMITTED_RESPONSES_TABLE_NAME         |                   |                                                                                               |
+| EQ_QUESTIONNAIRE_STATE_TABLE_NAME         |                   |                                                                                               |
+| EQ_SESSION_TABLE_NAME                     |                   |                                                                                               |
+| EQ_USED_JTI_CLAIM_TABLE_NAME              |                   |                                                                                               |
+| EQ_NEW_RELIC_ENABLED                      | False             | Enable New Relic monitoring                                                                   |
```

Comment:
Should EQ_NEW_RELIC_CONFIG_FILE also be here?

Reply:
Not used so ill remove